package.json

```diff
@@ -30,7 +30,7 @@
     "compile"
   ],
   "dependencies": {
-    "dustjs-linkedin": "^2.4.0",
+    "dustjs-linkedin": "^2.5.0",
```

Comment:
@kevva, @sethkinast is right as we're dependent on new functionality and would never want it to install an older version when deduped.